packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -6,6 +6,7 @@ import { Portal } from 'react-portal';
 import KeyCode from 'keycode-js';
 import Popup from 'terra-popup';
 import classnames from 'classnames';
```

Comment:
Shouldn't this line be removed, since it is replaced with `import classNames from 'classnames/bind';`